src/room/GroupCallLoader.tsx

```diff
@@ -51,7 +51,7 @@ export function GroupCallLoader({
   if (loading) {
     return (
       <FullScreenView>
-        <h1>{t("Loading room…")}</h1>
+        <h1>{t("Loading...")}</h1>
```

Comment:
Please use the `…` Unicode character rather than 3 dots, for consistency with our other strings